### ISSUE TITLE
add "email" to API call

### ISSUE THREAD

**Issue by hpiwowar** [ropensci/roadoi#1]
Thanks so much for making this R wrapper, it's fantastic!  !!!  yay :)

When you get a chance, could you add "email" as a param to the call signature and your usage example, and email=EMAIL to the api request?  Tracking the oadoi (via roadoi) usage allows us to get in touch if something breaks, and also means we can report stats to our funders, which helps us keep our non-profit afloat. :)  Thanks!

**Comment by njahn82** [comment]
Many thanks for your great tool!  I added email as a parameter. Hope it works for you

**Comment by njahn82** [comment]
Hi @hpiwowar I just want to make sure that the added email parameter and user agent are working for you before submitting it to CRAN. Thanks!

**Comment by jasonpriem** [comment]
Hey, sorry I'm late the party. Yep those are working great! Thanks for asking. Just one request:

We completely rebuilt a lot of oaDOI over the last month in order to deal with the massive traffic we were getting. As part of that, we found that is was actually easier to scale (using Heroku Dynos) if we got one DOI per request, using the GET endpoint. That's reflected in the new API docs, which no longer mention the POST. It's also for the most part easier for people to use.

So, if you get the chance or have the interest, we'd love if this wrapper could use the GET endpoint as well. 

Thanks again for making this...it's so cool! And we've used it ourselves a number of times now, as we work on improving our precision and recall. It's a great, easy way to get quick data on how good we're doing on various subsets of the literature.
